src/IonDecimal.ts

```diff
@@ -17,7 +17,22 @@ import {_sign} from "./util";
 
 /**
  * This class provides the additional semantics necessary for decimal values.
- * Note that range of an exponent is limited to +/- 15 digits.
+ *
+ * Unlike IEEE-754 floating point representation, Decimal values can represent numeric values without loss of precision.
+ * They combine an arbitrarily large integer value (the "coefficient") with a second integer representing its scale
```

Comment:
Most definitions of "scale" are "the number of digits to the right of the decimal point."  It's a bit different than exponent.

Reply:
I wasn't aware that 'scale' had an official meaning in this space, but that makes sense. I'll rephrase it.